### DISCUSSION TITLE
shadows

### DISCUSSION THREAD

**Discussion by bobblkabb** [hyprwm/Hyprland#5167] [Q&A] [answered]
Hi,

any chance to make a hyprland desktop like this? qtile:
![qtile](https://github.com/hyprwm/Hyprland/assets/7815101/cd70f19e-b47d-4ff7-bf2f-241f4e280a7c)

**Comment by fufexan** [comment]
Yes. See [`decoration:drop_shadow`](https://wiki.hyprland.org/Configuring/Variables/#decoration) for window shadows. For the bar, you should use your bar's method of setting shadows. For example, in Waybar/Eww/Ags/Ironbar you would use the `box-shadow` CSS property.

**Reply by foobar-p5** [reply]
on waybar it doesnt seem to work the right way, as it doesn't go outside the main window and is literally stuck inside. that can also be seen when rounded corners are present

**Reply by ghost** [reply]
> on waybar it doesnt seem to work the right way, as it doesn't go outside the main window and is literally stuck inside. that can also be seen when rounded corners are present

im having the same issue, did u find a way?

i can make the shadow appear but in return it will make a huge space below the bar 

wofi does the same thing and ive got around it by rendering it as a normal window, if waybar can do the same along with hyprland allowing me to display shadows on specific windows this will work

**Comment by bobblkabb** [comment]
But this is a "side shadow", it's only on 2 sides.

**Reply by fufexan** [reply, marked as answer]
`decoration:shadow_offset`. It's right there in the wiki.

**Reply by bobblkabb** [reply]
Thank you!
